tests/io/test.xpi.js

```diff
@@ -356,7 +356,7 @@ describe('Xpi.getChunkAsBuffer()', function getChunkAsBufferCallback() {
   it('should resolve with a buffer', () => {
     const myXpi = new Xpi('foo/bar', this.fakeZipLib);
     myXpi.files = {
-      'install.rdf': installRdfEntry,
+      'file_to.read': installFileEntry,
     };
```

Comment:
Same here, let's use `manifest.json` to be more real-world like. I am aware that it doesn't matter here but having our tests to use real-life names makes it easier to read in the future